src/services/twilio-api/twilio_api_v2010.json

```diff
@@ -10597,7 +10597,7 @@
             }
           },
           {
-            "description": "The pattern on which to match phone numbers. Valid characters are `*`, `0-9`, `a-z`, and `A-Z`. The `*` character matches any single digit. For examples, see [Example 2](https://www.twilio.com/docs/phone-numbers/api/availablephonenumberlocal-resource?code-sample=code-find-phone-numbers-by-number-pattern) and [Example 3](https://www.twilio.com/docs/phone-numbers/api/availablephonenumberlocal-resource?code-sample=code-find-phone-numbers-by-character-pattern). If specified, this value must have at least two characters.",
+            "description": "The pattern on which to match phone numbers. Valid characters are `*`, `0-9`, `a-z`, and `A-Z`. The `*` character matches any single digit. If specified, this value must have at least two characters. See [Examples](https://www.twilio.com/docs/phone-numbers/api/availablephonenumberlocal-resource?code-sample=code-find-phone-numbers-by-number-pattern)",
```

Comment:
These are generated files right, If we change something here will it not overwrite in future by` api-definitions`?

Reply:
Yes possible, there's no description field in api-definitions as such and the basic [json](https://code.hq.twilio.com/twilio/api-definitions/tree/603e2cf5e50d08c55c465654c14f1d0309de21f0/base/autopilot/v1) hasn't been modified a lot, last modified was 3 yrs ago. I tried finding the source of `description` and found [changes](https://github.com/twilio/twilio-cli-core/pull/117) on this repo itself to include it.

Reply:
Okay. Then It's fine.

Reply:
Thanks for confirming @shwetha-manvinkurke, I'll raise a PR for these changes on api-definitions.